src/GSF.CarbonIntensity/src/Handlers/ForecastHandler.cs

```diff
@@ -0,0 +1,59 @@
+using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.Exceptions;
+using CarbonAware.Tools.WattTimeClient;
+using CarbonAware.Tools.WattTimeClient.Configuration;
+using GSF.CarbonIntensity.Exceptions;
+using GSF.CarbonIntensity.Models;
+using Microsoft.Extensions.Logging;
+
+namespace GSF.CarbonIntensity.Handlers;
+
+internal sealed class ForecastHandler : IForecastHandler
+{
+    private readonly ICarbonAwareAggregator _aggregator;
+    private readonly ILogger<ForecastHandler> _logger;
+
+    public ForecastHandler(ILogger<ForecastHandler> logger, ICarbonAwareAggregator aggregator)
+    {
+        _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        _aggregator = aggregator ?? throw new ArgumentNullException(nameof(aggregator));
+    }
+
+    public async Task<EmissionsForecast?> GetCurrentAsync(string location, DateTimeOffset? start, DateTimeOffset? end, int? duration)
```

Comment:
Tested with JSON and WattTime data source.

The GetCurrentAsync command of the C# Library does not take an array or list of Locations as is required. The Location parameter is a string.

Reply:
@esgraham , great point. Will fix it.

Reply:
Bug fix pushed in